app/models/document.rb

```diff
@@ -22,4 +22,8 @@ def document_type_schema
   def newly_created?
     self.created_at == self.updated_at
   end
+
+  def created_by
```

Comment:
Have you considered setting up a `belongs_to :creator` relationship for this? It would prevent us from having to manually define the `created_by`.